athina/steps/base.py

```
@@ -81,6 +90,45 @@ def extract_input_data(self, context: Dict[str, Any]) -> Any:
             input_data = context
         return input_data
 
+    def _create_step_result(
+        self,
+        status: Literal["success", "error"],
+        data: str,
+        start_time: float,
+        metadata: Dict[str, Any] = {},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)